source/features/prevent-link-loss.tsx

```diff
@@ -17,19 +17,27 @@ import {
 } from '../github-helpers/prevent-link-loss.js';
 import createBanner from '../github-helpers/banner.js';
 
+const fieldSelector = [
+	'textarea.js-comment-field',
+	'textarea[aria-labelledby="comment-composer-heading"]', // Commit input field
```

Comment:
Commit? Or comment? Maybe it can just be 
```suggestion
	'textarea[aria-labelledby="comment-composer-heading"]', // React view
```